Hermes/appscale/hermes/stats/producers/proxy_stats.py

```diff
@@ -229,6 +228,8 @@ class ProxyStats(object):
   backend = attr.ib(metadata={Meta.ENTITY: HAProxyBackendStats})
   servers = attr.ib(metadata={Meta.ENTITY_LIST: HAProxyServerStats})
   listeners = attr.ib(metadata={Meta.ENTITY_LIST: HAProxyListenerStats})
+  servers_count = attr.ib()  # number of servers serving this proxy
+  listeners_count = attr.ib()  # number of listeners serving this proxy
```

Comment:
I see. Works for me.

If it's just a matter of responding differently depending on client requests, I think it would be simpler for the server to just call `len` on the list since it's a constant time operation. If it needs to be a class attribute, it could be defined in `__getattr__`. That way, whatever is using the object doesn't need to be responsible for keeping the attributes in sync.